core/src/main/java/nl/nn/adapterframework/senders/JavascriptSender.java

```diff
@@ -67,10 +68,18 @@ public class JavascriptSender extends SenderSeries {
 
 	private String fileInput;
 
+
 	public enum JavaScriptEngines {
-		J2V8(J2V8.class), NASHORN(Nashorn.class), RHINO(Rhino.class);
+		J2V8(J2V8.class),
+
+		@ConfigurationWarning("Engine Nashorn is deprecated. Use J2V8 instead.")
+		@Deprecated NASHORN(Nashorn.class),
+
+		@ConfigurationWarning("Engine Rhino is deprecated. Use J2V8 instead.")
+		@Deprecated RHINO(Rhino.class);
```

Comment:
```suggestion
		@Deprecated NASHORN(Nashorn.class),

		@Deprecated RHINO(Rhino.class);
```
Just the deprecated tag is enough here to 'hide' the element, the ConfigurationWarning should be on the setter element instead.